web/profiles/sdd_core/config/install/system.theme.yml

```diff
@@ -1,2 +1,2 @@
 admin: seven
-default: bartik
+default: seven
```

Comment:
Default is seven now?

Reply:
Yes, cause bartik looks broken with panelseverywhere)